docs/amqp-1-0-source.md

```diff
@@ -82,7 +83,8 @@ You can create a configuration file (JSON or YAML) to set the properties as belo
             "port": "5672",
             "username": "guest",
             "password": "guest",
-            "queue": "user-op-queue"
+            "queue": "user-op-queue",
+            "sessionMode": 2
```

Comment:
Shall we change to the default value `1`?

Reply:
Do you want us to add an extra test for sessionMode 2?

Reply:
Here is in docs and I guess users generally use the default value `1`? Or `2` is often preferred?

Reply:
For our use case we prefer that messages are only acked to amq-broker, when succesfully delivered to pulsar. We follow an at lease once principle for al our functions. We consider an implementation with sessionmode 2 an improvement.

Reply:
As we set the default `sessionMode` to `1` in code, and we have already docs about the new config option, I'd prefer to revert these lines about setting `sessionMode` to `2` but letting users to decide if they want to use a difference config. WDYT?

Reply:
@ronaldbuit What do you think also revert the extra `sessionMode` lines in docs changes?

Reply:
I removed it from the examples. Default mode is AUTO_ACKNOWLEDGE.